democracy/migrations/0013_auto_20160502_1222.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please use the `-n` flag with `makemigrations` to specify the migration's name. (`auto` migration names are not very nice.)